core/esmf-aspect-meta-model-interface/src/main/java/org/eclipse/esmf/metamodel/vocabulary/SAMM.java

```diff
@@ -106,6 +106,10 @@ public Resource Constraint() {
       return resource( "Constraint" );
    }
 
+   public Property constraint() {
+      return property( "constraint" );
+   }
+
```

Comment:
Please remove, since there is no `samm:constraint` (only `samm:Constraint` and `samm-c:constraint` which both already exist in the vocabulary classes).